### PR TITLE
[9.0] Port module res_partner_attributes_add_NEQ

### PR DESCRIPTION
**Pull request by JulieSFL**

### CODE REVIEW

File: res_partner_attributes_add_NEQ/i18n/nb_NO.po [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+#. module: res_partner_attributes_add_NEQ
+#: model:ir.model,name:res_partner_attributes_add_NEQ.model_res_partner
+msgid "Partner"
+msgstr "Partner"
+
+#. module: res_partner_attributes_add_NEQ
+#: help:res.partner,neq:0
+msgid "Quebec Enterprise Number (10 digits)"
+msgstr "Québec-firmanummer (10 sifre)"
```
Comment (ghost) [inline res_partner_attributes_add_NEQ/i18n/nb_NO.po:34]: @JulieSFL the translations dont make sense

File: res_partner_attributes_add_NEQ/i18n/nb_NO.po [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * res_partner_attributes_add_NEQ
+# 
+# Translators:
+# Imre Kristoffer Eilertsen <imreeil42@gmail.com>, 2016
```
Comment (ghost) [inline res_partner_attributes_add_NEQ/i18n/nb_NO.po:6]: who is this ?

File: res_partner_attributes_add_NEQ/i18n/nb_NO.po (hunk 1 above)
Comment (JulieSFL) [reply res_partner_attributes_add_NEQ/i18n/nb_NO.po:34]: @dufresnedavid Do you mean it makes no sense that this module has Norwegian translations?

**Review by ghost — COMMENTED** [review]
@JulieSFL never mind my comment

**Comment by ghost** [comment]
@joaoalf Travis seems to fail because it uses an old version of pip. This could explain why it is not able to find num2words. Do you have access to update it ?

**Comment by joaoalf** [comment]
@dufresnedavid It was a problem with the num2words wheel. They solved on their side. See https://github.com/savoirfairelinux/num2words/issues/53

**Comment by Ehtaga** [comment]
Good :+1:

**Review by jbchouinard — APPROVED** [review]

**Review by Ehtaga — APPROVED** [review]

**Comment by ghost** [comment]
:+1:

**Pull request merged**